src/databricks/labs/ucx/assessment/azure.py

```diff
@@ -33,6 +40,27 @@ class AzureServicePrincipalInfo:
     storage_account: str
 
 
+@dataclass
+class AzureSubscription:
+    name: str
+    subscription_id: str
+    tenant_id: str
+
+
+@dataclass
+class AzureStorageAccount:
+    name: str
+    resource_id: str
+    subscription_id: str
+
+
+@dataclass
+class AzureStorageSpnPermissionMapping:
+    storage_acct_name: str
```

Comment:
What about containers? What about ACLs on paths? It doesn't have to be done in this PR, but make data structure scale for changes.

Do we want to store this in SQL or CSV? Csv is editable via workspace, in case we want to decouple from az cli runs

Don't use abbreviations.